.github/workflows/label.yml

```
@@ -0,0 +1,27 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: PR Labeler
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_max_size: '100'
+          s_max_size: '500'
+          m_max_size: '1000'
```

```suggestion
          s_max_size: '250'
          m_max_size: '500'
```
What do you think? I took a quick look at Giving PRs and seems like a lot of components are in the 200-300 range.